docs/self-managed/setup/upgrade.md

```diff
@@ -561,230 +496,82 @@ webModeler:
       url: "jdbc:postgresql://web-modeler-postgres-ext:5432/rest-api-db"
 ```
 
-## From Camunda 8.1 to 8.2
-
-### Helm Chart 8.2.9
-
-#### Optimize
+</TabItem>
 
-For Optimize 3.10.1, a new environment variable introduced redirection URL. However, the change is not compatible with Camunda Helm charts until it is fixed in 3.10.3 (and Helm chart 8.2.9). Therefore, those versions are coupled to certain Camunda Helm chart versions:
+</Tabs>
 
-| Optimize version                  | Camunda Helm chart version |
-| --------------------------------- | -------------------------- |
-| Optimize 3.10.1 & Optimize 3.10.2 | 8.2.0 - 8.2.8              |
-| Optimize 3.10.3                   | 8.2.9+                     |
+## Upgrade Camunda
 
-No action is needed if you use Optimize 3.10.3 (shipped with this Helm chart version by default), but this Optimize version cannot be used out of the box with previous Helm chart versions.
+If you have installed the Camunda 8 Helm charts before with default values, Identity and the related authentication mechanism are enabled. If you have disabled Identity, see how to upgrade Camunda [with Identity disabled](#identity-disabled).
 
-### Helm Chart 8.2.3
+### Identity enabled
 
-#### Zeebe Gateway
-
-:::caution Breaking change
+#### Extract secrets
 
-Zeebe Gateway authentication is now enabled by default.
+If not specified on installation, the Helm chart generates random secrets for all components (including Keycloak). To upgrade successfully, these secrets must be extracted and provided during your upgrade.
 
-:::
-
-To authenticate:
-
-1. [Create a client credential](/guides/setup-client-connection-credentials.md).
-2. [Assign permissions to the application](/self-managed/identity/user-guide/authorizations/managing-resource-authorizations.md).
-3. Connect with:
-
-- [Desktop Modeler](/components/modeler/desktop-modeler/connect-to-camunda-8.md).
-- [Zeebe client (zbctl)](/self-managed/zeebe-deployment/security/secure-client-communication.md#zbctl).
-
```

Comment:
I believe I originally added the above section because I wanted customers to validate that their upgrade was successful, and when I'm testing upgrades, I often test by doing those steps. I'd hope to see this content again in another form, if not specified here.

Deploying a model covers scenarios where all the applications pods status report healthy, but maybe theres some communication issue between pods.

Reply:
Is this something that is useful to validate/test for every upgrade? Right now it's removed because it was only listed in the 8.1 -> 8.2 upgrade section, which is removed here because 8.1 is no longer supported. If a validation section would be useful in general, I can make another issue to add something to this guide for it.